immutable_tree.go

```diff
@@ -324,12 +323,7 @@ func (t *ImmutableTree) clone() *ImmutableTree {
 //
 
 func (t *ImmutableTree) nodeSize() int {
```

Comment:
nit: out of scope of this pr but returning 64 or 32 should be set here.

Reply:
it is only used in the test, if specify 32 or 64, we would have an inconvenience.